config/autoload/routes.global.php

```diff
@@ -147,6 +150,33 @@
             'allowed_methods' => ['GET', 'POST'],
         ],
         [
+            'name' => 'account-locations-list',
+            'path' => '/account/locations',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\ListLocationsAction::class,
+            ],
+            'allowed_methods' => ['GET'],
+        ],
+        [
+            'name' => 'account-location-add',
+            'path' => '/account/location/add',
+            'middleware' => [
+                App\Middleware\Authentication::class,
+                App\Action\Account\Location\AddLocationAction::class,
+            ],
+            'allowed_methods' => ['GET', 'POST'],
+        ],
+        [
+            'name' => 'account-location-edit',
+            'path' => '/account/location/{uuid}/edit',
```

Comment:
Yeah true the regex is quite long, it would prevent 500 errors though when it's invalid.

Interesting question about speed, I wonder if the regex would be validated after the other parts of the route match?